### ISSUE TITLE
build_wasm actions are failing with a pydantic error

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#2678]
e.g. https://github.com/sourmash-bio/sourmash/actions/runs/5512489542/jobs/10049386734?pr=2673 on PR #2673:

```
 pydantic.errors.PydanticUserError: If you use `@root_validator` with pre=False (the default) you MUST specify `skip_on_failure=True`. Note that `@root_validator` is deprecated and should be replaced with `@model_validator`.

For further information visit https://errors.pydantic.dev/2.0.2/u/root-validator-pre-skip
```

cc @luizirber

**Comment by ctb** [comment]
attempting fix in #2673 commit 0608088e996 with forcing pydantic < 2, per https://github.com/numpy/numpy/pull/24091 via https://github.com/numpy/numpy/issues/24088.

**Comment by ctb** [comment]
Yep fixed!
